app/routes/core.rb

```diff
@@ -125,6 +125,10 @@ def tracks
           ExercismWeb::Presenters::Tracks.tracks
         end
 
+        def active_tracks
+          X::Track.all.select(&:active)
```

Comment:
let's cache this with  `@active_tracks ||=`

At the moment I'm going to have to restart the app anyway if we activate new languages. I'll think about how to deal with that differently when I start tackling overall performance stuff.